airflow/requirements.txt

```diff
@@ -1,4 +1,4 @@
-apache-airflow-providers-weaviate==1.0.0
+apache-airflow-providers-weaviate==1.3.0rc2
```

Comment:
Looks like [1.3.0](https://airflow.apache.org/docs/apache-airflow-providers-weaviate/stable/index.html) is there. Should we use it instead?

Reply:
Done